addons/web/static/tests/legacy/mobile/views/fields/many2one_barcode_field_tests.js

```diff
@@ -41,7 +41,7 @@ QUnit.module("Fields", (hooks) => {
                         {
                             id: 111,
                             name: "product_cable_management_box",
-                            barcode: "601647855631",
+                            barcode: "6016478556318",
```

Comment:
Why this change ?

Reply:
Alright but this is not a demo data, it's a data used by QUnit tests only, I don't think it makes a difference to update them

Reply:
Alright, I thought they are affected by demo data since they have the same exact barcode